src/entities/video-contents/hooks/usePlaylist.ts

```
@@ -0,0 +1,64 @@
+import { withdrawcontentsApi } from "@entities/withdraw-recommends/api";
+import { PlaylistSource } from "@shared/types";
+import { PlaylistResponse } from "@shared/types";
+import { bookmarkApi } from "@/entities/bookmark/api";
+
+export const parsePlaylistSource = (
+  searchParams: URLSearchParams,
+): PlaylistSource => {
+  const playlist = searchParams.get("playlist");
+  switch (playlist) {
+    case "topTag":
+      const rawIndex = Number(searchParams.get("index"));
```

Biome (2.4.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)